.github/workflows/update_formula.yml

```diff
@@ -0,0 +1,57 @@
+name: Homebrew update formula
+on:
+  workflow_dispatch:
+    inputs:
+      AM_VERSION:
+        required: true
+        description: "am release version (just numbers without the v, e.g.: 2.2.0)"
+        type: string
+      SHA256_AARCH64_APPLE_DARWIN:
+        required: true
+        description: "SHA256 for macOS arm64 binary"
+        type: string
+      SHA256_X86_64_APPLE_DARWIN:
+        required: true
+        description: "SHA256 for macOS x86 binary"
+        type: string
+      SHA256_X86_64_LINUX_GNU:
+        required: true
+        description: "SHA256 for Linux x86 binary"
+        type: string
+      SHA256_AARCH_64_LINUX_GNU:
+        required: true
+        description: "SHA256 for Linux x86 binary"
+        type: string
+    secrets:
+      PRIVATE_GITHUB_TOKEN:
```

Comment:
I think we should be able to use the built-in token, see: https://github.com/actions/checkout#push-a-commit-using-the-built-in-token

Reply:
👍 updated!